shared_assets/shader/distance_field_sprite_fill.pixel.metal

```diff
@@ -36,6 +36,7 @@ fragment float4 pixel_func (
 	float edge1 = clamp(inEdge + s, edge0 + 0.01, 0.99);
 	float edge = smoothstep(edge0, edge1, a) * v.colour.a;
 
+	float4 colFill = v.colour;
```

Comment:
I think you can just refer to `v.colour` in the below lines rather than introducing a new variable

Reply:
Personal opinion is that it's a bit easier to read this way + is closer to the format in other shader files.

If you feel strongly about this, I can put the change in.

Reply:
Don't feel strongly; compiler should optimise it out anyway